service-front/assets/sass/patterns/_accordion.scss

```diff
@@ -108,7 +108,7 @@
 
         .link-edit {
             display: block;
-            padding: $gutter-half $gutter;
+            padding: $gutter-half $gutter-half;
```

Comment:
What was the reason for this, does it affect a lot of pages? Presumably yes? Is it at all dangerous in terms of breaking anything else? Hopefully no. ;)

Reply:
It affects all accordions for editing. But because they should all be using the same macro, and therefore all get the altered text ("Change" instead of "edit" or "Edit"), they will all need to use this adjusted CSS.